pkg/virt-operator/kubevirt.go

```diff
@@ -701,6 +703,8 @@ func (c *KubeVirtController) execute(key string) error {
 		}
 	}
 
+	configuration.SetConfigurationMetrics(kv)
```

Comment:
I think it won't be enough to calculate it only here, since user can patch the configuration during "runtime", so you probably need to calculate the metric each time a new config update occur, please look [here](https://github.com/enp0s3/kubevirt/blob/ef62d0f87a95580c2c449fa4bbd1c5a13d15d81a/pkg/virt-config/configuration.go#L320)

Reply:
the `func (c *KubeVirtController) execute(key string) error` is called to each reconcile loop, so it would catch any configuration patch

Reply:
@machadovilaca You are correct, for some reason I thought you put it in `cmd` in the `main` function. Please ignore

Reply:
This is suboptimal. It's not the best place to have it here and it will be called more than it should. `virt-config` does have a callback that can be used for this.

Reply:
updated